en/docs/Learn/DesignAPI/CreateAPI/expose-a-soap-service-as-a-rest-api.md

```diff
@@ -1,63 +1,67 @@
-# Expose a SOAP service as a REST API
+# Expose a SOAP Service as a REST API
 
 WSO2 API Manager supports the management of an existing SOAP and WSDL based services exposing as REST APIs.
-The organizations who have SOAP/ WSDL based services, can easily bridge their existing services to REST without the cost of a major migration. WSO2 API Manager supports two kinds of services as one for performing a  "pass through" of the SOAP message to the backend and other one is generating [a RESTful api from the backend SOAP service](../../../../Learn/DesignAPI/CreateAPI/generate-rest-api-from-soap-backend/).
+The organizations that have SOAP/WSDL based services can easily bridge their existing services to REST without the cost of a major migration. WSO2 API Manager supports two types of services - one for performs a pass-through of the SOAP message to the backend and the other generates [a RESTful API from the backend SOAP service]({{base_path}}/Learn/DesignAPI/CreateAPI/generate-rest-api-from-soap-backend/).
```

Comment:
one for performs a pass-through should be corrected to **one performs a pass-through**

Reply:
Resolved with https://github.com/wso2/docs-apim/pull/646/commits/4d069b8ae975df3b38f9e9afde8bd582a52dd5c6